.github/workflows/iroha2-dev-pr-bench.yml

```diff
@@ -0,0 +1,29 @@
+name: I2::Dev::Bench
```

Comment:
In general I think that CI integration should be done in a separate PR by a DevOps. Running benchmarks on the CI runners is a bad idea, since the runners are "ondemand", meaning that the performance can vary. I would remove this workflow file entirely and focus on the actual benchmark.

Reply:
Yeah, I'm going to remove this workflow together with `tps-ci` on my final commit in that case